search_gov_crawler/elasticsearch/convert_html_i14y.py

```diff
@@ -0,0 +1,73 @@
+import os
+import hashlib
+import newspaper
+from datetime import datetime, timezone
+from urllib.parse import urlparse
+
+def convert_html(html_content: str, url: str, domain_name: str):
+    article = newspaper.article(url=url, input_html=html_content)
+    article.nlp()
+    description_en = article.meta_description or article.summary
+    content_en = article.text_cleaned or article.text or description_en or article.title
+
+    if content_en is None:
+        return None
+
+    time_now_str = current_utc_iso()
+    path = article.url or url
+
+    basename, extention = get_base_extension(url)
+    sha_id = generate_url_sha256(path)
+
+    i14y_doc = {
+        "audience": None,
+        "changed": None,
+        "click_count": None,
+        "content_type": None,
+        "created_at" : time_now_str,
+        "created" : None,
+        "_id": sha_id,
+        "id" : sha_id,
+        "thumbnail_url" : article.meta_img or article.top_img,
+        "language" : article.meta_lang or "en",
+        "mime_type" : "text/html",
+        "path" : path,
+        "promote" : None,
+        "searchgov_custom1" : None,
+        "searchgov_custom2" : None,
+        "searchgov_custom3" : None,
+        "tags" : article.tags or article.keywords or article.meta_keywords,
+        "updated_at" : time_now_str,
+        "updated" : None,
+        "title_en" : article.title,
```

Comment:
This should match the language. e.g. in Chinese
```
 "title_chs" : article.title
```

Reply:
Good catch! What should my backup be if the language is undetermined? I noticed we also have fields: `title` and `content` can I use those (if we don't know the language)?